src/main/java/seedu/momentum/model/project/Project.java

```diff
@@ -41,6 +57,22 @@ public Name getName() {
         return Collections.unmodifiableSet(tags);
     }
 
+    /**
+     * Returns an unmodifiable view of the durations list.
+     */
+    public ObservableList<WorkDuration> getDurationList() {
+        return durations.asUnmodifiableObservableList();
+    }
+
+    /**
+     * Adds a WorkDuration to the duration list.
+     * Duplicates will not be added.
+     * @param duration
+     */
+    public void addDuration(WorkDuration duration) {
+        durations.add(duration);
+    }
+
```

Comment:
By below I mean in the `Project.java` class, can't really highlight it from here.

Reply:
This is mostly what i was referring to when i said more/better tests are needed.

Basically, since I'm using LocalDateTime.now() to get the current time, 2 times will never be exactly the same (they will be some nanoseconds apart). For example in tests, even if I start 2 timers immediately after one another, they will have different values, so equals() will still say that they're different. Since the existing tests involve checking if 2 models are equal, this will break a bunch of existing tests.

I'm think of either (a) Overloading the timer functions so that I can start them at a fixed time (just for testing) or (b) Introduce a custom global clock for the application to run on. The global clock will be the same as system clock in production, but in testing it will have functions that allow us to manually control the time.